src/app/utils/__tests__/date.spec.ts

```diff
@@ -67,4 +71,23 @@ describe('Date Util', () => {
       })
     })
   })
+
+  describe('getStartOfDay', () => {
+    it(`should return today's start of day if date is not given`, () => {
+      // Act
+      const result = getStartOfDay()
+      // Assert
+      const today = new Date(new Date().setHours(0, 0, 0, 0))
```

Comment:
Tip: use `timezone-mock` package to mock timezones! Can refer to my code testing some date utils in `frontend/src/utils/date.test.ts`

Reply:
I don't think i need to mock the timezone for this function?